### ISSUE TITLE
[NPC][Quest] With a Little Help...

### ISSUE THREAD

**Issue by MrTide** [SallyWhitemane/Maelstrom#248]
**Description:** Quest targets got too much HP

**How to reproduce:** NPCS have wrong health values
![image](https://user-images.githubusercontent.com/36106057/174488134-167de3f6-3303-4758-8059-b32286d4442a.png)
![image](https://user-images.githubusercontent.com/36106057/174488166-8ec485bf-7fb5-483a-ab3f-284fb8c07e4b.png)
![image](https://user-images.githubusercontent.com/36106057/174488174-7c4212f0-db57-4ffe-a24c-f9527287e21a.png)

**How it should work:** they should have hps listed below
https://cata-twinhead.twinstar.cz/?npc=41076
https://cata-twinhead.twinstar.cz/?npc=41081
https://cata-twinhead.twinstar.cz/?npc=41082
basically 3,302 health for each
